index.html

```diff
@@ -35,6 +39,7 @@
          don’t hesitate and contact me.</p></div>
        <div class="rectangle">
           <div>Start collaboration</div></div>
```

Comment:
Please ensure that the `Start collaboration` be a button according to the requirement.